packages/playwright/src/runner/testServer.ts

```diff
@@ -117,6 +123,12 @@ class Dispatcher {
     await this._runTests(params.reporter, params.locations, params.options);
   }
 
+  async findRelatedTestFiles(params:  { files: string[] }): Promise<ConfigFindRelatedTestFilesReport> {
+    const config = await this._loadConfig({});
+    const runner = new Runner(config);
+    return runner.findRelatedTestFiles('out-of-process', params.files);
```

Comment:
I wonder how this compiles, when `runner.findRelatedTestFiles` may return `{ errors }`, but `ConfigFindRelatedTestFilesReport` has non-optional `testFiles`.